packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -194,14 +201,15 @@ class GoogleSignInPlugin extends GoogleSignInPlatform {
 
     if (currentUser == null) return false;
 
-    final grantedScopes = currentUser.getGrantedScopes();
+    final grantedScopes = currentUser.getGrantedScopes() ?? '';
     final missingScopes =
         scopes.where((scope) => !grantedScopes.contains(scope));
 
     if (missingScopes.isEmpty) return true;
 
-    return currentUser
-            .grant(auth2.SigninOptions(scope: missingScopes.join(" "))) ??
-        false;
+    final response = await currentUser
+        .grant(auth2.SigninOptions(scope: missingScopes.join(" ")));
```

Comment:
Nit: `' '`

Reply:
I've swept the code for unneeded double-quoted strings. Good catch!